### PR TITLE
Map player list hud

### PR DESCRIPTION
**Pull request by enbrain**

### CODE REVIEW

File: mappings/net/minecraft/client/gui/hud/PlayerListHud.mapping [hunk 1]
```diff
@@ -1,30 +1,54 @@
...
+	FIELD field_32209 HALF_HEART_U I
+	FIELD field_32210 GOLDEN_HEART_U I
+	FIELD field_32211 HALF_GOLDEN_HEART_U I
+	FIELD field_32212 BLINKING_HEART_U I
+	FIELD field_32213 BLINKING_HALF_HEART_U I
 	METHOD <init> (Lnet/minecraft/class_310;Lnet/minecraft/class_329;)V
 		ARG 1 client
 		ARG 2 inGameHud
 	METHOD method_1918 getPlayerName (Lnet/minecraft/class_640;)Lnet/minecraft/class_2561;
+		COMMENT {@return the player name which is rendered by this HUD}
```
Comment (liach) [inline mappings/net/minecraft/client/gui/hud/PlayerListHud.mapping]:
```suggestion
		COMMENT {@return the player name rendered by this HUD}
```

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/client/gui/hud/PlayerListHud.mapping [hunk 2]
```diff
@@ -1,30 +1,54 @@
...
+	FIELD field_32209 HALF_HEART_U I
+	FIELD field_32210 GOLDEN_HEART_U I
+	FIELD field_32211 HALF_GOLDEN_HEART_U I
+	FIELD field_32212 BLINKING_HEART_U I
+	FIELD field_32213 BLINKING_HALF_HEART_U I
 	METHOD <init> (Lnet/minecraft/class_310;Lnet/minecraft/class_329;)V
 		ARG 1 client
 		ARG 2 inGameHud
 	METHOD method_1918 getPlayerName (Lnet/minecraft/class_640;)Lnet/minecraft/class_2561;
+		COMMENT {@return the player name rendered by this HUD}
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/hud/PlayerListHud.mapping:29]:
```suggestion
		COMMENT @return the player name rendered by this HUD
```

File: mappings/net/minecraft/client/network/PlayerListEntry.mapping [hunk 3]
```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_640 net/minecraft/client/network/PlayerListEntry
-	FIELD field_3738 health I
+	FIELD field_3736 health I
+	FIELD field_3737 lastHealthTime J
+	FIELD field_3738 lasthealth I
```
Comment (YanisBft) [inline mappings/net/minecraft/client/network/PlayerListEntry.mapping]:
```suggestion
	FIELD field_3738 lastHealth I
```

File: mappings/net/minecraft/client/gui/hud/PlayerListHud.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/client/gui/hud/PlayerListHud.mapping:29]: See https://github.com/FabricMC/yarn/pull/2442#discussion_r641437146

**Review by YanisBft — APPROVED** [review]

**Review by LambdAurora — APPROVED** [review]

**Pull request merged**